stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -461,11 +490,12 @@ function externDecryptAesCbc(byte[] input, byte[] key, byte[] iv, handle padding
 } external;
 
 # Returns AES ECB decrypted value for the given AES ECB encrypted data.
+# ```ballerina byte[]|crypto:Error plainText = crypto:decryptAesEcb(cipherText, key); ```
 #
 # + input - The content to be decrypted
 # + key - Encryption key
 # + padding - The padding
-# + return - Decrypted data or `Error` if key is invalid
+# + return - Decrypted data or `crypto:Error` if key is invalid
```

Comment:
```suggestion
# + return - Decrypted data or `crypto:Error` if key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.